benchmark/assert/throws.js

```diff
@@ -13,7 +13,7 @@ function main({ n, method }) {
   const doNotThrowError = () => { return 'foobar'; };
   const regExp = /foobar/;
   const message = 'failure';
-  var i;
+  let i;
```

Comment:
nit: move this to loop inside each case where it's used?
i.e. lines 23, 30 and 37
```
for (let i = 0; i < n; ++i) {
```